### PR TITLE
[CUP-2929] Allow scope and tuum personID to be returned on access token

### PR DESCRIPTION
**Pull request by RorySimpson**

### CODE REVIEW

File: src/services/tokenGenerator.ts [hunk 1]
```diff
@@ -150,7 +160,11 @@ export class JwtTokenGenerator implements TokenGenerator {
       event_id: eventId,
       iat: authTime,
       jti: uuid.v4(),
-      scope: "aws.cognito.signin.user.admin", // TODO: scopes
+      scope: getScopeForClient(userPoolClient),
+      "custom:tuum_person_id": attributeValue(
+        "custom:tuum_person_id",
+        user.Attributes
+      ),
```
Comment (ben-clarke) [inline src/services/tokenGenerator.ts:164]: Are you able to pass in as config?

Comment (RorySimpson) [reply src/services/tokenGenerator.ts:164]: scopes now passed in

File: src/services/tokenGenerator.ts [hunk 2]
```diff
@@ -85,12 +85,9 @@ const applyTokenOverrides = (
 };
 
 const BASE_SCOPE = "aws.cognito.signin.user.admin";
-const BANKING_SERVICES_USER_SCOPES =
-  "capability-platform/person capability-platform/savings capability-platform/document connect/read capability-platform/payments authentication-service/saltedge verification-service/idv authentication-service/user capability-platform/statements capability-platform/idv capability-platform/account-information authentication-service/user-write connect/write capability-platform/account-management authentication-service/verify";
-
 const getScopeForClient = (client: AppClient): string => {
-  if (client.ClientName === "banking-services-app-client")
-    return `${BASE_SCOPE} ${BANKING_SERVICES_USER_SCOPES}`;
+  if (client.Scopes)
```
Comment (ben-clarke) [inline src/services/tokenGenerator.ts:89]: If I was being picky I would say only add the base scope on if it was initiated using AdminInitiateAuth

Comment (ben-clarke) [reply src/services/tokenGenerator.ts:89]: But it'll do

**Review by ben-clarke — APPROVED** [review]

**Pull request merged**